SYCL/Sampler/unnormalized-none-linear.cpp

```diff
@@ -0,0 +1,182 @@
+// RUN: %clangxx -fsycl -fsycl-targets=%sycl_triple %s -o %t.out
+// RUN: %HOST_RUN_PLACEHOLDER %t.out %HOST_CHECK_PLACEHOLDER
+// RUN: %GPU_RUN_PLACEHOLDER %t.out %GPU_CHECK_PLACEHOLDER
+
+// CPU failing at moment. Waiting on fix.
```

Comment:
is it possible to run the test on CPU but XFAIL it to detect the moment when the problem is resolved?